### ISSUE TITLE
Add osrcry

### ISSUE THREAD

**Issue by benjaminoakes** [benjaminoakes/maid#118]
https://github.com/geemus/osrcry

Might help with the release process

**Comment by coaxial** [comment]
Could be used along with GH Actions. Or maybe there are already existing actions for this? Tracking in #211, #212

**Comment by benjaminoakes** [comment]
https://github.com/maid/maid/issues/212#issuecomment-1475275950

**Comment by geemus** [comment]
Yeah, I think an action makes a lot of sense. There aren't existing ones that I'm aware of. I used to just have it as part of my rake based release process and/or I would at least try to remember to run it manually (neither of which is ideal). Just let me know if you have questions or anything around it.

**Comment by benjaminoakes** [comment]
👍

Thanks @geemus!
